test/Libraries/CoreNodesTests/StringTests.cs

```diff
@@ -288,7 +288,7 @@ public static void AllIndicesOf()
             Assert.AreEqual(new[] {1, 4}, String.AllIndicesOf("mississippi", "is"));
         }
 
-        [Test]
+        [Test, Category("FailureNET6")]
```

Comment:
@sm6srw this seems to be because of the behavior change for `string.lastindexof` in .net 5, if I'm not wrong. See https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/lastindexof-improved-handling-of-empty-values. Can you verify? If so, seems to be an easy fix and we can either:
1. Update the test, document change in behavior to users
2. If we wish to maintain previous behavior, we could treat the empty string case as a special case in our own `string.lastindexof` node.
What do you think?

Reply:
I'm ok with putting this off as a separate task as well.

Reply:
yes, that is what's going on. We need to agree on the behavior change. I will defer this as well.